main.go

```diff
@@ -25,29 +25,43 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+var termSignalChan = make(chan os.Signal, 1)
+
 func init() {
 	// SIGUSR1 forces an update check.
 	// SIGUSR2 forces an inventory update.
 	// SIGTERM marks the exit.
 	signal.Notify(cli.SignalHandlerChan, syscall.SIGUSR1, syscall.SIGUSR2, syscall.SIGTERM)
```

Comment:
We should remove `syscall.SIGTERM` from here. I guess it is harmless (not covered in if/else if block) but clearer if removed.

Reply:
(if we do this, the patch below would require adjustment)

Reply:
Well spotted, you're right, I'll fix it!